Code/GraphMol/MMPA/Wrap/testMMPA.py

```diff
@@ -33,21 +33,21 @@ def test1(self):
 
     fs = Chem.GetMolFrags(frags[0][1], asMols=True)
     self.assertEqual(len(fs), 2)
-    self.assertEqual(Chem.MolToSmiles(fs[0], True), 'c1ccc([*:1])cc1')
-    self.assertEqual(Chem.MolToSmiles(fs[1], True), 'CO[*:1]')
+    self.assertEqual(Chem.MolToSmiles(fs[0], True), 'c1ccc([*:2])cc1')
```

Comment:
I'm not sure I understand why the canonicalization ends up resulting in `:2` instead of `:1`, but if you're comfortable that's right I can live with it.